maproulette/src/main/java/com/pixeltron/mapquest/open/geocoding/RequestMaker.java

```diff
@@ -0,0 +1,35 @@
+package com.pixeltron.mapquest.open.geocoding;
+
+public class RequestMaker {
+
+    private Request geoReq, fsReq;
+
+    public RequestMaker() {
```

Comment:
It might be a good option to add a private `RequestParameters` that is set in the constructor, as it would make sense that any request that would be made will require it (and is the case for both the geocoding and foursquare). This would allow for passing in at construction time rather than requiring it to be passed in for each type of request. This would have the `RequestMaker` class be more stateful, rather than simply a proxy object for the underlying request builders.